src/react/databrowser/objects/details/ObjectRestorationButtonAndModal.tsx

```diff
@@ -0,0 +1,166 @@
+import { ChangeEvent, useState } from 'react';
+import {
+  Banner,
+  Icon,
+  Modal,
+  PrettyBytes,
+  Stack,
+  Text,
+  Wrap,
+} from '@scality/core-ui';
+import { Button, Input } from '@scality/core-ui/dist/next';
+import { useMutation } from 'react-query';
+import { useSelector } from 'react-redux';
+import { AWSError } from 'aws-sdk';
+import { getClients } from '../../../utils/actions';
+import { AppState } from '../../../../types/state';
+import { formatShortDate } from '../../../utils';
+
+type RestoreObjectType = {
+  bucketName: string;
+  objectKey: string;
+  objectLastModifiesOn: string;
+  objectSize: number;
+  objectStorageClass: string;
+  isObjectRestoredOrOnGoing: boolean;
+  objectVersionId?: string;
+};
+const ObjectRestorationButtonAndModal = ({
+  bucketName,
+  objectKey,
+  objectLastModifiesOn,
+  objectSize,
+  objectStorageClass,
+  isObjectRestoredOrOnGoing,
+  objectVersionId,
+}: RestoreObjectType) => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [restorationDays, setRestorationDays] = useState(1);
+  const [restoreError, setRestoreError] = useState('');
+  const { zenkoClient } = getClients(useSelector((state: AppState) => state));
+  const handleClose = () => {
+    setIsModalOpen(false);
+  };
+
+  const ObjectTable = () => {
+    return (
+      <table style={{ borderCollapse: 'collapse' }}>
+        <thead>
+          <tr style={{ borderBottom: '1px solid #323245' }}>
```

Comment:
```suggestion
          <tr style={{ borderBottom: `1px solid ${IMPORT_THIS_COLOR_FROM_THE_THEME_WITH_useTheme()}` }}>
```

Reply:
nit:  do we _really_ want to use  bgLevel2 here ? We also have the `border` color